mautic/mautic
# [UX] Humanized time display in Twig extension

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR allows us to have a time display like "2 hours ago", "3 minutes ago" or "just now".
Until now, we had only this format "X days ago" for dates, but not time.






---
### 📋 Steps to test this PR:


1. Pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the file `app/bundles/CoreBundle/Resources/views/Notification/notification.html.twig`
3. Change `{{ dateToText(n.dateAdded) }}` to `{{ dateToHumanized(n.dateAdded) }}`
4. Go to Contacts > Page actions > Export to CSV to trigger a notification
5. Reload the page
6. Open the Notifications panel
7. See the humanized time
